collect_app/src/main/java/org/odk/collect/android/geo/GoogleMapFragment.java

```diff
@@ -271,6 +271,17 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
         return featureId;
     }
 
+    @Override
+    public List<Integer> addMarkers(List<MarkerDescription> markers) {
```

Comment:
For the moment OSM and Google just call `addMarker` repeatedly from `addMarkers`. I'm not sure if either have the ability to have a similar implementation to Mapbox, but I didn't want to spend time on that now given I was focused on the crash.

Reply:
I remember that at least in OSM it is possible to add a collection of markers. Could you file an issue for that?